### PR TITLE
Force users to select an autocomplete value

### PR DESCRIPTION
**Pull request by Manoj-nathwani**
See [Jira ticket](https://fjelltopp.atlassian.net/browse/ADX-394?atlOrigin=eyJpIjoiNDhjY2I0ODUxYmQxNGQxNjkwZThjMjVjMmJhOGY3ZjkiLCJwIjoiaiJ9)

### CODE REVIEW

File: package_schemas/general_data/2_restricted_schema.json [hunk 1]
```diff
@@ -116,12 +116,12 @@
                 }
             ],
 	    "form_attrs_users": {
-		"data-module": "autocomplete",
+    "data-module": "autocomplete-without-creating-new-options",
```
Comment (tomeksabala) [inline package_schemas/general_data/2_restricted_schema.json]: fix indents please

File: package_schemas/general_data/2_restricted_schema.json [hunk 2]
```diff
@@ -116,12 +116,12 @@
...
 		"data-module-tags": "",
 		"data-module-source": "/api/2/util/user/autocomplete?q=?"
             },
 	    "form_attrs_orgs": {
-		"data-module": "autocomplete",
+    "data-module": "autocomplete-without-creating-new-options",
```
Comment (tomeksabala) [inline package_schemas/general_data/2_restricted_schema.json]: fix indents please

**Review by tomeksabala — CHANGES_REQUESTED** [review]

**Review by jonathansberry — APPROVED** [review]
Yep with the indents fixed I'm happy for this to be merged.  It should be merged once we have already merged the other PR though.

**Comment by Manoj-nathwani** [comment]
Ok this is all done ✅ 
Will wait for https://github.com/fjelltopp/ckanext-unaids/pull/67 to progress before merging

**Pull request merged**